ETC.tsv

```diff
@@ -97,109 +97,109 @@ ETC_20150317_000096	$Although their bright color makes them pretty visible, it's
+ETC_20150317_000174	$Leaves fallen from a dead Hanaming wither rapidly.
 ETC_20150317_000175	$Chupacabra
-ETC_20150317_000176	Their numbers grew after the Great Plant Cataclysm; now they tend to compete with humans for territory.
+ETC_20150317_000176	$Their numbers grew after the Great Plant Cataclysm; now they tend to compete with humans for territory.
 ETC_20150317_000177	$Gray Chupacabra
 ETC_20150317_000178	$Large Gray Chupacabra
-ETC_20150317_000179	Chupacabras move secretly. Based on this, it is more dangerous for humans if there are elite Chupacabras.
+ETC_20150317_000179	$Chupacabras move secretly. Based on this, it is more dangerous for humans if there are elite Chupacabras.
 ETC_20150317_000180	$Yellow Chupacabra
 ETC_20150317_000181	$Popolion
-ETC_20150317_000182	Some say a Popolion's voice can even break glass bottles.
+ETC_20150317_000182	$Some say a Popolion's voice can even break glass bottles.
 ETC_20150317_000183	$Sauga
 ETC_20150317_000184	$Saugas
 ETC_20150317_000185	$Scorpion
 ETC_20150317_000186	$Tontulia
-ETC_20150317_000187	It is a mutated monster for Tontulia to be more active.
+ETC_20150317_000187	$the Tontus mutated into Tontulia to acquire mobility.
 ETC_20150317_000188	$Tontus
-ETC_20150317_000189	The lives of residents in the dry and harsh environment were further impacted when the cacti mutated to Tontus.
+ETC_20150317_000189	$As cacti mutated into Tontus the lives of the residents in dry and harsh environments worsened.
 ETC_20150317_000190	$Vekarabe
 ETC_20150317_000191	$Compared to other animal species, Vekarabes didn't change much from their original form when they became the monsters. However, their aggresion is the exception.
 ETC_20150317_000192	$Geppetto
-ETC_20150317_000193	The name of this monster suits its appearance, as it doesn't have any reflection.
+ETC_20150317_000193	$The name of this monster suits its appearance, but it doesn't reflect its nature at all.
 ETC_20150317_000194	$Boowook
-ETC_20150317_000195	One of the defensive systems left in the king's tomb.
+ETC_20150317_000195	$One of the defensive systems left by Zachariel the Great in the Royal Mausoleum.
 ETC_20150317_000196	$Bearkaras
 ETC_20150317_000197	$Pino
-ETC_20150317_000198	Early pioneers came up with its name from its similarity to the Gepetto.
+ETC_20150317_000198	$The name stuck because the first people to discover Geppeto and Pino thought they shared similarities.
 ETC_20150317_000199	$Helmet Bug
-ETC_20150317_000200	Monsters that have simply grown in size and become more aggressive are rather relieving.
+ETC_20150317_000200	$Monsters that have simply grown in size and become more aggressive are rather relieving.
 ETC_20150317_000201	$Cauliflower
-ETC_20150317_000202	This stationary monster is especially hostile towards humans, but it will still attack all warm-blooded animals.
+ETC_20150317_000202	$Although this stationary monster isn't especially hostile towards humans, it will still attack all warm-blooded animals.
```

Comment:
"Though this stationary monster isn't especially hostile towards humans, it still attacks all warm-blooded animals."

Just a bit of a rewrite here.